i18n/es.jsonp.js

```diff
@@ -83,14 +84,14 @@ document.localeJson = {
       "configure-compatibility-helpstub": "para una explicación de la diferencia entre estos modos:",
       "configure-compatibility-option-serviceworker": "<strong>Modo ServiceWorker</strong> (<i>recomendado</i>, soporta contenido dinámico y ejecución arbitraria de JS)",
       "configure-compatibility-option-serviceworker-tip": "Este modo requiere que el navegador o el framework sean capaces de instalar un Service Worker. Permite contenido ZIM dinámico y ejecutar cualquier elemento con JavaScript sin restricciones. Funciona interceptando las llamadas Fetch del navegador y suministrando el contenido solicitado desde el ZIM.",
-      "configure-compatibility-option-serviceworkerlocal": "<strong>ServiceWorkerLocal</strong> (Service Worker local seguro, pero <i>sin ejecución arbitraria de JavaScript</i>)",
-      "configure-compatibility-option-serviceworkerlocal-tip": "[Sólo para extensiones de Chromium:] Este modo ejecuta el Service Worker en un sandbox local seguro, y no necesita acceso único a nuestro servidor PWA. Sin embargo, debido a las fuertes restricciones de seguridad en la API, no es capaz de ejecutar todos los fragmentos sueltos de JavaScript, por lo que el contenido ZIM dinámico puede fallar.",
-      "configure-compatibility-option-jquery": "<strong>JQuery</strong> (<i>obsoleto</i>: no soporta contenido dinámico, compatible con navegadores antiguos)",
-      "configure-compatibility-option-jquery-tip": "Este modo está obsoleto, pero funciona bien con los archivos de Wikipedia / WikiMedia. Funciona en todos los dispositivos.",
+      "configure-compatibility-option-serviceworkerlocal": "<strong>ServiceWorkerLocal</strong> (Service Worker local Seguro, pero <i>sin ejecución arbitraria de JavaScript</i>)",
+      "configure-compatibility-option-serviceworkerlocal-tip": "[Sólo para extensiones de Chromium:] Este modo ejecuta el Service Worker en un sandbox local Seguro, y no necesita acceso único a nuestro servidor PWA. Sin embargo, debido a las fuertes restricciones de seguridad en la API, no es capaz de ejecutar todos los fragmentos sueltos de JavaScript, por lo que el contenido ZIM dinámico puede fallar.",
+      "configure-compatibility-option-jquery": "<strong>Seguro Modo</strong> (No soporta contenido dinámico, compatible con navegadores antiguos)",
```

Comment:
```suggestion
      "configure-compatibility-option-serviceworkerlocal": "<strong>ServiceWorkerLocal</strong> (Service Worker local seguro, pero <i>sin ejecución arbitraria de JavaScript</i>)",
      "configure-compatibility-option-serviceworkerlocal-tip": "[Sólo para extensiones de Chromium:] Este modo ejecuta el Service Worker en un sandbox local seguro, y no necesita acceso único a nuestro servidor PWA. Sin embargo, debido a las fuertes restricciones de seguridad en la API, no es capaz de ejecutar todos los fragmentos sueltos de JavaScript, por lo que el contenido ZIM dinámico puede fallar.",
      "configure-compatibility-option-jquery": "<strong>Modo Seguro</strong> (No soporta contenido dinámico, compatible con navegadores antiguos)",
```
The word `seguro` ins ome of these isn't referring to the mode, but is just the adjective "secure", and so shouldn't be capitalized. Also, Safe Mode is "Modo Seguro", not "Seguro Modo".